### PR TITLE
[DDW-621] Improve currency Japanese translations

### PR DESCRIPTION
**Pull request by daniloprates**
This PR CHANGES.

## Todos

- [x] Add the `ja` key to the currencies JSON list
- [x] Change the code to use the `en`/`ja` keys instead of `name`
- [x] Add the Japanese translation - (**Junko**)

## Screenshots

Screen Shot 2021-04-09 at 16 26 46
Screen Shot 2021-04-09 at 16 26 50

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test
---

## Testing scenarios

Scenario 1 - Check currencies according to language
```gherkin
Given that Daedalus selected language is |English/Japanese|
And I am on the "Wallets" screen from "Settings"
And the option of show balance in other currencies is enabled
When I check the names of the currencies in the dropdown
Then I can see that the names are displayed in the selected language
```

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by gabriela-ponce — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM 👍 . Tested on 17573.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Changelog
 
 ### Features
 
+- Added Japanese translation to the external currencies list ([PR 2497](https://github.com/input-output-hk/daedalus/pull/2497))
+- Added rewards address in Rewards Delegation Screen ([PR 2475](https://github.com/input-output-hk/daedalus/pull/2475))
```
Comment (tomislavhoracek) [inline CHANGELOG.md:29]: - _Added Japanese translation to the external currencies list ([PR 2497](https://github.com/input-output-hk/daedalus/pull/2497))_ - This should be moved to the **vNext**
- _Added rewards address in Rewards Delegation Screen ([PR 2475](https://github.com/input-output-hk/daedalus/pull/2475))_ - This should be removed

**Review by tomislavhoracek — CHANGES_REQUESTED** [review]
@daniloprates Please fix CHANGELOG and improve PR description.
Other changes are good!

**Review by nikolaglumac — APPROVED** [review]
💯

File: CHANGELOG.md (hunk 1 above)
Comment (nikolaglumac) [reply CHANGELOG.md]: Fixed in https://github.com/input-output-hk/daedalus/pull/2497/commits/c457640f73269722b63352452754b33689b55dd3

**Pull request merged**